src/main.py

```diff
@@ -0,0 +1,64 @@
+import graph
+import pygame
+import draw
+import env
+
+DISPLAY_WIDTH = 800
+DISPLAY_HEIGHT = 600
+PATH = env.get_graph_data_path()
+
+description = """
+  Find of optimal paths by the Minty method
+  by Dmytro Prystaichuk
+  """
+
+nodes_coords = {0 : (70, 210),
```

Comment:
Dict, where keys are integers, is literally just an array, so you could just use `list` here

Reply:
In future, I want add ability to put node name using letter